examples/bri-3/src/bri/workgroup/workflows/models/workflow.ts

```diff
@@ -14,16 +15,24 @@ export class Workflow {
   @AutoMap()
   workgroupId: string;
 
+  @AutoMap()
+  bpiAccountId: string;
+
+  @AutoMap(() => BpiAccount)
+  bpiAccount: BpiAccount;
+
   constructor(
     id: string,
     name: string,
     worksteps: Workstep[],
     workgroupId: string,
+    bpiAccount: BpiAccount,
```

Comment:
@ognjenkurtic should this not also contain the `bpiAccountId`?

Reply:
Good point. Usage of both bpiAccountId and bpiAccount (and this pattern is used across the codebase whenever there is a reference to an object) is not necessary. It is a convention we picked up from how prisma.schema objects are defined. I added a commit that updates bpiAccountId just in the ctor just to be consistent, will have to change this as part of a separate issue that i ll open now.

Reply:
https://github.com/eea-oasis/baseline/issues/719